Assignment/Test.c

```diff
@@ -7,8 +7,26 @@
 // Program on main branch.
 // ------------------------------------------------------------------------------------------------
 #include "Program.h"
+#include <stdio.h>
+
+void DisplayRes (ComplexNumber num) {
+   printf ("%.2f + %.2fi\n", num.real, num.img);
+}
 
 int main () {
-   PrintHelloWorld ();
+   ComplexNumber num1, num2, res1, res2, res3;
+   printf ("enter real part 1 and img part 1:");
+   scanf_s ("%f ", &num1.real);
+   scanf_s ("%f", &num1.img);
+   printf ("\nenter real part 2: ");
+   scanf_s ("%f", &num2.real);
+   printf ("\nenter img part 2:");
+   scanf_s ("%f", &num2.img);
+   res1 = Op_Add (num1, num2);
+   DisplayRes (res1);
+   res2 = Op_Sub (num1, num2);
+   DisplayRes (res2);
+   res3 = Op_mul (num1, num2);

```

Comment:
Naming should be consistent. For add and sub, you have capitalized the operation (e.g. Op_Add) but for multiply it is op_mul. This is not elegant.